packages/nectar/src/sagas/appConnection/connection.slice.test.ts

```diff
@@ -43,4 +69,38 @@ describe('connectionReducer', () => {
     )
     expect(registrars).toEqual({ [registrarId]: true })
   })
+
+  it('add/remove connected peerId from store and get it correctly', () => {
```

Comment:
Maybe there's a way to use RTL to check the exact nectar's sagas behavior, instead of manually dispatching actions only for test purposes?